shared/constants/form.ts

```diff
@@ -11,7 +11,7 @@ const PUBLIC_FORM_FIELDS = <const>[
   'title',
   '_id',
   'responseMode',
-  'payments',
+  'payments_field',
```

Comment:
should we restrict the `payments_field` to be accessible only by storage mode forms?